importanize/main.py

```diff
@@ -122,6 +129,23 @@ def run_importanize(path, config, args):
     text = read(path)
     file_artifacts = get_file_artifacts(path)
 
+    # Get formatter from args or importanizerc
+    if args.formatter:
+        config['formatter'] = args.formatter
+    elif config.get('formatter'):
+        pass
+    else:
+        config['formatter'] = 'IndentWithTabsFormatter'
+    # Check formatter availibility or exit
+    try:
```

Comment:
not a fan of `try .. except`. I like the way Django template tags are registered so maybe do something more explicit:

```python
from importanize import formatters

# next to PEP8_CONFIG above
FORMATTERS = {
    formatter.name: formatter
    for formatter in vars(formatters).values()
    if (inspect.isclass(formatter)
        and formatter is not formatters.Formatter
        and issubclass(formatter, formatters.Formatter))
}

# in --formatter argument
# this will prevent users using bad formatter since then argparse will blow up due to choices=..
parser.add_argument(
    '-f', '--formatter',
    type=six.text_type,
    default=None,
    choices=sorted(FORMATTERS.keys()),
    help='Formatter used.'
)

# here
formatter = FORMATTERS.get(args.formatter or config.get('formatter'), DEFAULT_FORMATTER)

# in each formatter class:
class FooFormatter(Formatter):
     name = 'foo'
     def format(self): pass
```

will allow use:

```bash
$ importanize --formatter=foo ...
```